style.css

```
@@ -0,0 +1,195 @@
+* {
+    box-sizing: border-box;
+}
+
+html {
+    font-family: Pretendard;
+}
+
+body {
+    margin: 0;
+}
+
+a {
+    text-decoration: none;
+    cursor: pointer;
+}
+
+h1 {
+    display: block;
+    margin: 0;
+    color: #374151;
+    font-size: 40px;
+    font-weight: 700;
+    line-height: 140%;
+}
+
+
+header {
+    height: 70px;
+    background-color: #FFFFFF;
+    border-bottom: 1px solid #dfdfdf;
+    padding: 0 200px;
+    justify-content: center;
+    align-content: center;
+}
+
+header div {
+    max-width: 1120px;
+    margin: 0 auto;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+header img {
+    height: 51px;
+}
+
+header .login {
+    width: 128px;
+    height: 48px;
+    border-radius: 8px;
+    background-color: #3692FF;
+    color: #F3F4F6;
+    font-size: 16px;
+    font-weight: 600;
+    text-align: center;
+    align-content: center;
+}
+
+
+.banner {
+    height: 540px;
+    background-color: #CFE5FF;
+    align-content: end;
+}
+
+.banner.bottom {
+    margin-top: 138px;
+}
+
+.banner > div {
+    width: 1110px;
+    margin: 0 auto;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.banner .banner-explain {
+    margin-bottom: 60px;
+}
+
+.banner a {
+    width: 357px;
+    height: 56px;
+    margin-top: 32px;
+    border-radius: 40px;
+    background-color: #3692FF;
+    color: #F9FAFB;
+    font-size: 20px;
+    font-weight: 600;
+    text-align: center;
+    align-content: center;
+    display: block;
+}
+
+.banner img {
+    width: 746px;
+}
+
+
+main {
+    background-color: #FFFFFF;
+}
+
+article {
+    height: 720px;
+    align-content: center;
+}
+
+article > div {
+    display: flex;
+    gap: 64px;
+    justify-content: center;
+    align-items: center;
+}
+
+.article-top {
+    color: #3692FF;
+    margin-bottom: 12px;
+    font-size: 18px;
+    font-weight: 700;
+}
+
+.article-explain {
+    color: #374151;
+    margin-top: 24px;
+    font-size: 24px;
+    font-weight: 500;
+    line-height: 133.33%;
```

💊 제안

해당 값이 어디서 나온 것인지 모르겠지만, 소수점 단위의 사용은 의도를 파악하기 어려울 수 있어 추천하지 않습니다.
참고로 숫자만으로도 비슷한 결과를 구현할 수 있습니다.
예를 들어, line-height: 1.3;은 폰트 크기의 1.3배를 의미하고, line-height: 133.333%;는 폰트 크기의 133%로 결과적으로 비슷한 효과를 냅니다.
가독성과 유지보수를 고려하면 아래와 같이 1.3 형태로 작성하는 것이 더 직관적이니 참고해 주세요~

```suggestion
    line-height: 1.3;
```